tests/framework/installer/ceph_installer.go

```diff
@@ -127,13 +129,26 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 		h.k8shelper.ChangeHostnames()
 	}
 
-	rookOperator := h.Manifests.GetRookOperator(namespace)
+	rookOperator := h.Manifests.GetRookOperator(namespace, h.EnableCSI)
 
 	_, err = h.k8shelper.KubectlWithStdin(rookOperator, createFromStdinArgs...)
 	if err != nil {
 		return fmt.Errorf("Failed to create rook-operator pod : %v ", err)
 	}
-
+	if h.EnableCSI {
+		csiTemplate := h.Manifests.GetCephCSI(namespace)
+		_, err = h.k8shelper.KubectlWithStdin(csiTemplate, createFromStdinArgs...)
+		if err != nil {
+			return fmt.Errorf("Failed to create cephcsi : %v ", err)
+		}
+	} else {
+		//create dummy configmap for csi-rbd-config and csi-cephfs-config
+		cm := &v1.ConfigMap{}
+		cm.Name = "csi-rbd-config"
```

Comment:
Seems like we could always create these configmaps as part of the common.yaml. If csi isn't enabled, they don't hurt anything.